receiver/oracledbreceiver/metadata.yaml

```diff
@@ -387,3 +387,131 @@ metrics:
       value_type: int
       input_type: string
     unit: "{executions}"
+  oracledb.query.application_wait_time:
```

Comment:
i still think it is confusing, they are listed under `metrics` but actually used in the `log` pipeline. i would suggest to not touch the metadata file for now until we got log support on mdatagen.